src/main/java/uk/gov/hmcts/reform/hmc/model/OrganisationDetails.java

```diff
@@ -19,7 +19,6 @@
     @Size(max = 60, message = ValidationError.ORGANISATION_TYPE_MAX_LENGTH)
     private String organisationType;
 
-    @NotEmpty(message = ValidationError.CFT_ORG_ID_NULL_EMPTY)
```

Comment:
We should be able to remove the constant from the `ValidationError` too with this PR, as it should no longer be used anywhere.

Reply:
removed

Reply:
updated in 0ae612a810d04011ea9c0e28759e449f3a4fa43d